test/dommock/DOMMockBuilder.ts

```diff
@@ -838,101 +838,12 @@ export default class DOMMockBuilder {
             // eslint-disable-next-line @typescript-eslint/no-explicit-any
             stat: (_name: string): any => 'stat1-value',
             id: 'RTCInboundRTPAudioStream',
-            timestamp: 1,
-            type: 'inbound-rtp',
             ssrc: 1,
-            kind: 'audio',
-            ...mockBehavior.rtcPeerConnectionGetStatsReport,
-          },
-          {
-            names: (): string[] => ['stat2'],
```

Comment:
Didn't you add this from the last PR?

Reply:
At that time I misunderstood the usage of these stuff. Although they are not wrong, but it does not obey the original idea. So I just revert these changes and improve the unit test.

Reply:
Nice :)